### PR TITLE
Type split improvements

### PR DESCRIPTION
**Pull request by h67ma**
After spending some time with PLTypeSplitSymbols, some quirks became annoying to me. This is an attempt to improve it (and some related layouts too).

Suggestions welcome. I'm fine with creating a new layout instead of modifying existing ones, if that's preferable.

Side note: it would be cool if the user was able to customize the layout without recompiling, using some sort of editor, or by loading a json/xml/whatever file.

### CODE REVIEW

File: app/src/main/java/com/dessalines/thumbkey/keyboards/CommonKeys.kt [hunk 1]
```diff
@@ -72,6 +72,40 @@ val ABC_OR_NUMERIC_SWIPES =
             ),
     )
 
+val SPECIAL_ACTIONS_SWIPES =
```
Comment (dessalines) [inline app/src/main/java/com/dessalines/thumbkey/keyboards/CommonKeys.kt:75]: Thx for abstracting these.

**Review by dessalines — COMMENTED** [review]
Seems fine to me, I don't use these typesplit keyboards, so its up to yall to come to a consensus on them.

Run `./gradlew formatKotlin` to pass lint.

**Review by dessalines — APPROVED** [review]

**Comment by kkard2** [comment]
> Side note: it would be cool if the user was able to customize the layout without recompiling, using some sort of editor, or by loading a json/xml/whatever file.
yes, it would be nice, because this just broke my muscle memory

**Comment by h67ma** [comment]
> > Side note: it would be cool if the user was able to customize the layout without recompiling, using some sort of editor, or by loading a json/xml/whatever file.
> 
> yes, it would be nice, because this just broke my muscle memory

Sorry about that. Honestly I didn't expect this would be merged so quickly and without any discussion.

**Comment by dessalines** [comment]
Either of you can discuss here and make fixes to it, and do a PR. I don't use those layouts so I don't know what changes aren't desirable.

**Comment by fililip** [comment]
> > Side note: it would be cool if the user was able to customize the layout without recompiling, using some sort of editor, or by loading a json/xml/whatever file.
> 
> yes, it would be nice, because this just broke my muscle memory

Same for me, I updated to the latest version and realized I can't type as easily anymore, had to downgrade.

**Comment by dessalines** [comment]
PRs welcome. No one so far has tried to fix these layouts.

**Pull request merged**